### PR TITLE
[feature1] feature1 added

### PR DESCRIPTION
**Pull request by ping2kalesh**
adds feature1

### CODE REVIEW

**Comment by suhailhd423** [comment]
please add one more line

**Comment by ping2kalesh** [comment]
Sure. I will do it

File: feature1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 This is new feature added as part of feature1
+new line added
```
Comment (suhailhd423) [inline feature1.txt:2]: looks good to me

**Review by suhailhd423 — APPROVED** [review]
I have reviewed and approving the PR

**Pull request merged**